### ISSUE TITLE
Line 1470 of paps.cc is causing problems with output passed to ps2pdf

### ISSUE THREAD

**Issue by larryhynes** [dov/paps#52]
Hi

Building paps from `HEAD`:

Line 1470 of `src/paps.cc` (`print("Got`` {} pages\n", num_pages);`) is putting that line, uncommented, at the head of the ps output, which leads `ps2pdf` to emit an error (`Error: /undefined in Got`). So the test case `echo 'test' | ./src/paps | ps2pdf - test.pdf` fails.

**Comment by larryhynes** [comment]
`echo 'test' | ./src/paps | head` gives the following output:

```ps
Got 1 pages
%!PS-Adobe-3.0
%%Creator: cairo 1.17.6 (https://cairographics.org)
%%CreationDate: Sun Jan 29 16:58:12 2023
%%Pages: 1
%%DocumentData: Clean7Bit
%%LanguageLevel: 2
%%DocumentMedia: A4 595 842 0 () ()
%%Title: stdin
%%Orientation: Portrait
```

I don't think that first line `Got 1 pages` should be there.

**Comment by dov** [comment]
Thanks. Fixed in commit sha1:58afdc8a7 on master.
